cordyceps/src/list.rs

```diff
@@ -158,6 +158,49 @@ impl<T: Linked<Links<T>> + ?Sized> List<T> {
         // tracing::trace!(?self, "push_front: pushed");
     }
 
+    /// Appends an item to the tail of the list
+    pub fn push_back(&mut self, item: T::Handle) {
+        let ptr = T::into_ptr(item);
+        // tracing::trace!(?self, ?ptr, "push_back");
```

Comment:
my guess is that these are here because they're in the `push_back` and `pop_front` methods, but because they're commented out, i have a slight preference for not adding them in new code.

we should maybe eventually add a feature flag to control whether tracing is enabled in `cordyceps`... but we should probably do that in a separate branch.

Reply:
Removed for this PR! :D